adafruit_lis3dh.py

```diff
@@ -120,7 +130,11 @@ def __init__(self, int1=None, int2=None):
             self._int1.pull = digitalio.Pull.UP
 
     @property
-    def data_rate(self):
+    def data_rate(
+        self,
+    ) -> Literal[
+        0, 0b0001, 0b0010, 0b0011, 0b0100, 0b0101, 0b0110, 0b0111, 0b1000, 0b1001
```

Comment:
This is technically correct, thought `int` may be clearer (albeit makes it seem like it can take any `int`).  Since this will make the documentation in RTD less readable, I'm leaning towards `int` for this and the setter.

Reply:
Tks see https://github.com/adafruit/Adafruit_CircuitPython_LIS3DH/pull/76/commits/59c669238f330e5703bb89641199c02303db38c7